src/opera/test/pge/dswx_ni/test_dswx_ni_pge.py

```diff
@@ -408,12 +426,12 @@ def test_dswx_ni_pge_validate_output_product_filenames(self):
         pge._validate_output_product_filenames()
 
         # Verify good test data before triggering errors
-        band_data = ('OPERA_L3_DSWx-NI_T12345_20210101T120000Z_20210101T120000Z_LSAR_30_v1.0_B01_WTR.tif',
-                     'OPERA_L3_DSWx-NI_T12345_20210101T120000Z_20210101T120000Z_LSAR_30_v1.0_B02_BWTR.tif',
-                     'OPERA_L3_DSWx-NI_T12345_20210101T120000Z_20210101T120000Z_LSAR_30_v1.0_B03_CONF.tif',
+        band_data = ('OPERA_L3_DSWx-NI_T11SLS_20210101T120000Z_20210101T120000Z_LSAR_30_v1.0_B01_WTR.tif',
```

Comment:
In this block, the date's not matching stands out.  I know it doesn't matter, so consider this another one of my nitty comments.